site2/tools/release-json-gen.sh

```diff
@@ -51,12 +51,12 @@ do
 done
 
 # gh auth login, ref: https://github.com/apache/pulsar-site/blob/760f7b47f437267a3c9d71d14c73e99ba32bd1ec/site2/website-next/scripts/release-notes.sh#L31-L33
-echo "$GH_TOKEN" > "$TOKEN_FILE"
+echo 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 > "$TOKEN_FILE"
```

Comment:
@urfreespace you should never store tokens in git. Please make sure to invalidate this token immediately.  /Cc @Anonymitaet @tisonkun @codelipenghui

Reply:
@urfreespace who is associated with this token, are you? Could you clarify what permissions are associated with this token?

We'd better replace all literal tokens, regardless of read-only or read-write, with GitHub secrets via INFRA team members. I believe they always respond in hours.

Reply:
https://github.com/apache/pulsar-site/pull/235 @tisonkun @lhotari removed PTAL